api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -557,6 +559,27 @@ public AuthenticationResponse cancel(ProceedContext proceedContext) {
         }
     }
 
+    /**
+     * Handle Polling.
```

Comment:
Should this method do the polling logic to detect when the remediation/authenticator has changed?

Reply:
This is still a debate item - should backend SDK do polling automatically or have the client app drive it. So for now, the SDK just exposes the polling API here.